_drafts/JSCON Budapest

```diff
@@ -0,0 +1,122 @@
+---
+
+title: 'JS CON  Budapest'
+author: Pawan
+layout: blog-post
+comments: true
+
+---
+
+Hey Everyone!
+
+The  Hoodies  (Jan, Lena, Nick, Nolan)  were  just  at  JSConf   Budapest  this   week ! It   was an exciting event with lots of cool stuff happening!
+
+# JSCON Budapest
+
+Situated in Budapest, this year's JavaScript conference showcased plenty of workshops and engaging speakers.
+Budapest is a city located on the Danube river. The capital consists of two cities, Buda and Pest.Budapest has been
+reported as one of the best places to visit in Europe. As such it makes for a perfect place to  hold the JS conference.
+
+#Speakers
+1.LAURIE VOSS @seldo
```

Comment:
Some of the names are in ALL CAPS, some are not. Could we make sure each speakers name is written in the same way?